ooiservices/app/models.py

```diff
@@ -807,24 +816,14 @@ def from_json(json):
 
     @staticmethod
     def insert_user(username='admin', password=None, first_name='First', last_name='Last', email='FirstLast@somedomain.com', org_name='ASA'):
-        user = User(user_id=username, email=email)
-        try:
-            user.validate_username(username)
-            user.validate_email(email)
-        except ValidationError as e:
-            admin_del = db.session.query(User).filter((User.user_name == username) | (User.email == email)).first()
-            if admin_del != None:
-                db.session.delete(admin_del)
```

Comment:
This will delete the existing user account, as 'user' would never have gotten to the db.